topics/authentication.md

```diff
@@ -6,140 +6,151 @@ Code examples: <a href="samples.md" anchor="authentication">Authentication</a>
 </p>
 </microformat>
 
-<include src="lib.xml" include-id="outdated_warning"/>
+Ktor provides the `Authentication` feature to handle authentication and access control. Typical usage scenarios include logging in users, granting access to specific resources, and secure transmitting information between parties. You can also use `Authentication` with [Sessions](sessions.md) to keep a user's information between requests.
```

Comment:
The header refers to authentication and authorization, and in the first sentence we refer to it as access control. It may be better to use the same terminology.

Reply:
and secure transmitting -> as well as securely transmitting